src/draw_spline_paths.js

```diff
@@ -150,17 +150,17 @@ dc_graph.draw_spline_paths = function(pathreader, pathprops, hoverprops, selectp
     }
 
     // convert original path data into <d>
-    function genPath(path, old, lineTension, avoidSharpTurn, angleThreshold) {
+    function genPath(originalPoints, lineTension, avoidSharpTurn, angleThreshold) {
       var c = lineTension || 0;
-      avoidSharpTurn = avoidSharpTurn !== false;
-      angleThreshold = angleThreshold || 0.02;
+      var avoidSharpTurn = avoidSharpTurn !== false;
+      var angleThreshold = angleThreshold || 0.02;
 
       // helper functions
-      var vecDot = function(v0, v1) { return v0.x*v1.x+v0.y*v1.y; };
-      var vecMag = function(v) { return Math.sqrt(v.x*v.x + v.y*v.y); };
+      var vecDot = function(v0, v1) { return v0.x*v1.x+v0.y*v1.y };
+      var vecMag = function(v) { return Math.sqrt(v.x*v.x + v.y*v.y) };
 
       // get coordinates
-      var path_coord = getNodePositions(path, old);
+      var path_coord = getNodePositions(originalPoints);
```

Comment:
The changes above look like they accidentally revert some changes I made - `genPath` now takes another argument that tells it whether to look at the old positions or new positions, so these changes may break the animations and/or `lineTension`.

Reply:
yeah, this is a mistake. Please don't take this part.